src/main/kotlin/dev/jamiecraane/gptmentorplugin/ui/chat/messages/ChatBubble.kt

```diff
@@ -33,20 +37,25 @@ class ChatBubble (msg : String, user : Boolean = true): JBPanel<ChatBubble>() {
     }
 
     fun createContentComponent(content :String) : Component {
+        val component = if (isUser) questionComponent else responseComponent
+        responseComponent.contentType = "text/html; charset=UTF-8"
+        responseComponent.updateMessage(content)
```

Comment:
if the application is currently handling an user's prompt then `responseComponent.updateMessage(content)` should not be called